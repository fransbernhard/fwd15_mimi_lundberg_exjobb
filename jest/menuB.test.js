import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import MenuB from '../src/components/menuB';

describe('components', () => {
  describe('<MenuB />', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
        <MenuB />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
