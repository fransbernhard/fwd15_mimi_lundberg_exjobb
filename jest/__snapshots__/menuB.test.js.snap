// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <MenuB /> renders correctly 1`] = `
<header>
  <nav>
    <ul>
      <li>
        <a
          href="#top"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="#about"
          onClick={[Function]}
        >
          About
        </a>
      </li>
      <li>
        <a
          onClick={[Function]}
          style={Object {}}
        >
          Archive
        </a>
      </li>
      <li
        className="contactMobile"
      >
        <a
          href="mailto:magdamargaretha@gmail.com?Subject=magdenmagden"
        >
          Contact
        </a>
      </li>
      <li
        className="contactWeb"
      >
        <a
          href="#contact"
          onClick={[Function]}
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
